src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -320,6 +324,7 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
       previewAs: this.previewAsPerson,
     }).pipe(finalize(() => this.isFeedbackSessionQuestionsLoading = false))
         .subscribe((response: FeedbackQuestionsResponse) => {
+          if (response.questions.length === 0) { this.isFeedbackSessionQuestionResponsesLoading = false; }
```

Comment:
Why it is set when the length is zero?

Reply:
Because then `this.loadFeedbackQuestionRecipientsForQuestion()` is never called